android/src/com/mozilla/vpn/VPNServiceBinder.kt

```diff
@@ -246,7 +253,10 @@ class VPNServiceBinder(service: VPNService) : Binder() {
         ifaceBuilder.parsePrivateKey(privateKey)
         ifaceBuilder.addAddress(InetNetwork.parse(jDevice.getString("ipv4Address")))
         ifaceBuilder.addAddress(InetNetwork.parse(jDevice.getString("ipv6Address")))
-        
+
+        ifaceBuilder.addDnsServer(DNS.mullvadVpnInternal)
```

Comment:
Don't do that. Instead use: jServer.getString("ipv4Gateway"). That corresponds to Server::Ipv4Gateway()